docs/.eleventy.js

```diff
@@ -407,6 +407,20 @@ module.exports = function(eleventyConfig) {
         }
     });
 
+    /*
+     * Generate the sitemap only in certain contexts to prevent unwanted discovery of sitemaps that
+     * contain URLs we'd prefer not to appear in search results (URLs in sitemaps are considered important).
+     * In particular, we don't want to deploy https://eslint.org/docs/head/sitemap.xml
+     */
+    if (!(
```

Comment:
Can we distribute the “not” here? I’m finding this condition a bit difficult to understand in its current form.

Reply:
I've refactored this a bit now, please check if the new version looks easier to reason about.